dspace/config/modules/identifiers.cfg

```diff
@@ -0,0 +1,43 @@
+#----------------------------------------------------------------------#
+#---------------------IDENTIFIER CONFIGURATIONS------------------------#
+#----------------------------------------------------------------------#
+# These configs are used for additional identifier configuration such  #
+# as the Show Identifiers step which can "pre-mint" DOIs and Handles   #
+#----------------------------------------------------------------------#
+
+# Should DOIs be minted for (future) registration at workspace item creation?
+# A DOI created at this stage will be in a 'PENDING' status while in workspace and workflow.
+# At the time of item install, the DOI filter (if any) will be applied and if the item matches the filter, the DOI
+# status will be updated to TO_BE_REGISTERED. An administrator can also manually progress the DOI status, overriding
+# any filters, in the item status page.
+# This option doesn't require the Show Identifiers submission step to be visible.
+# Default: false
+#identifiers.submission.register = true
+
+# This configuration property can be set to a filter name to determine if a PENDING DOI for an item
+# should be queued for registration. If the filter doesn't match, the DOI will stay in PENDING or MINTED status
+# so that the identifier itself persists in case it is considered for registration in the future.
+# See doi-filter and other example filters in item-filters.xml.
+# Default (always_true_filter)
+#identifiers.submission.filter.install = doi-filter
+
+# This optional configuration property can be set to a filter name, in case there are some initial rules to apply
+# when first deciding whether a DOI should be be created for a new workspace item with a PENDING status.
+# This filter is only applied if identifiers.submission.register is true.
+# This filter is updated as submission data is saved.
+# Default: (always_true_filter)
+#identifiers.submission.filter.workspace = always_true_filter
+
+# If true, the workspace filter will be applied as submission data is saved. If the filter no longer
+# matches the item, the DOI will be shifted into a MINTED status and not displayed in the submission section.
+# If false, then once a DOI has been created with PENDING status it will remain that way until final item install
+# Default: true
+#identifiers.submission.strip_pending_during_submission = true
+
+# This configuration property can be set to a filter name to determine if an item processed by RegisterDOI curation
+# task should be eligible for a DOI
+#identifiers.submission.filter.curation = always_true_filter
+
+# Show Register DOI button in item status page?
+# Default: false
+#identifiers.item-status.register = true
```

Comment:
Shouldn't this be named specific to DOIs if it only works for DOIs?  So something like `identifiers.item-status.registerDOI = true`?

Reply:
yes good call, this should help to make things more clear.